070.mediawiki

```diff
@@ -0,0 +1,51 @@
+<pre>
+BUIP070: Support BitPay's new Bitcoin Cash address format in BUCash
+Proposer: Gal Buki
```

Comment:
please add also your bitcoi.in forum handle between brackets (ditto for all the other pre tag)